vulcanai/models/metrics.py

```diff
@@ -91,7 +92,7 @@ def get_score(targets, predictions, metrics='accuracy', average=None,
         return results_dict
 
     @staticmethod
-    def transform_outputs(in_matrix, transform_callable=None, **kwargs):
+    def transform_outputs(in_matrix):
         """
         Reformat output matrix.
 
```

Comment:
I think this function description will need to be updated now that transform_callable no longer plays a role here

Reply:
oops, will do

Reply:
fixed